CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ STATE ENCRYPTION
 
 NEW FEATURES:
 * Add support for a `removed` block that allows users to remove resources or modules from the state without destroying them. ([#1158](https://github.com/opentofu/opentofu/pull/1158))
+* Provider defined functions are now available.  They may be referenced via `provider::provider_alias::funcname(args)`.  ([#1439](https://github.com/opentofu/opentofu/pull/1439))
```

Comment:
To clarify - do I understand correctly that provider::funcname(args) also works?

Reply:
It does not, they are only available via provider aliases

Reply:
Huh, would it make sense for them to also be available without them? Similar to how it works for resources.

Reply:
How would you handle conflicts?

Reply:
Same way as it's currently handled for resources, whatever that way is.

Reply:
Alright, I misunderstood provider as being a placeholder, in that case, could we clarify by rewriting this to `provider::<provider_alias>::<funcname>(args)` or something like that, to  emphasize what is and what is not a placeholder here?